spec/features/photogram_2016_spec.rb

```diff
@@ -1,30 +1,50 @@
 require "rails_helper"
 
-# / && /PHOTOS
-feature "Home page (same as /photos)" do
-  scenario "Home page has an h1 tag with text 'List of Photos' (should always pass since initial repo takes care of this)", points: 1 do
-    visit "/"
-    expect(page).to have_selector("h1", text: "List of Photos")
+feature "READ: " do
+  scenario "photos#index RCAV works", points: 1 do
+    visit "/photos"
+
+    expect(page).to have_selector("h1")
   end
-end
 
-# /PHOTOS/:ID
-feature "Photo details page (/photos/:id)" do
-  photo = FactoryGirl.create(:photo)
+  scenario "photos#index displays a list of existing photos", points: 1 do
+    first_photo = FactoryGirl.create(:photo)
+    second_photo = FactoryGirl.create(:photo)
```

Comment:
On second thought, this feels Mystery Guest-ish. We should just include the attributes in question in line here, e.g.

```ruby
create(:photo, source: "http://source.of/first_photo.png", caption: "First photo")
```